x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/EqlFunctionRegistry.java

```diff
@@ -32,15 +37,20 @@ public EqlFunctionRegistry() {
         // Scalar functions
         // String
             new FunctionDefinition[] {
+                def(Add.class, Add::new,"add"),
```

Comment:
Do I need to wrap these into surrogatefunction derived class or can I use them directly?

Reply:
I didn't even realize you could do this, so this removes a lot of boilerplate.
I think `divide` will be the only one where we'll want a surrogate

Reply:
yeah, was looking to write less boilerplate code, but wanted to double check with @costin if this appropriate in the grand scheme of things

Reply:
I would add them directly (that's what we do in SQL). `SurrogateFunction` is useful when a function results in a chain of other functions, otherwise it can be a subclass or like here, the base class itself.